src/lib/location.ts

```diff
@@ -47,19 +49,19 @@ export async function resolveLocation(
   } else {
     const { address } = zipOrAddress;
     const response = await geocoder.geocode(address, ['census2020']);
-    const result = response?.results?.at(0);
+    const result = response.results.at(0);
 
     if (!result || result.address_components.country !== 'US') {
       return null;
     }
 
+    // @ts-expect-error census.2020 is not included in the typedef
     const censusInfo = result.fields.census['2020'];
```

Comment:
I kept the expect-error directive here because the `Census` type does not have a `2020` property according to the type definitions.

Reply:
Should it be? It [looks like it's a valid key](https://www.geocod.io/docs/#census-block-tract-fips-codes-amp-msa-csa-codes), and one of the contributors to the thread [here](https://github.com/Geocodio/geocodio-library-node/issues/17#issuecomment-2470818477) offered some type definitions for `Census`.